clients/rust/src/client.rs

```diff
@@ -44,6 +45,12 @@ impl ClientBuilder {
             .no_deflate()
             .no_gzip()
             .no_zstd()
+            // The read timeout "applies to each read operation", so should work fine for larger
+            // transfers that are split into multiple chunks.
+            // We define both as 500ms which is still very conservative, given that we are in the same network,
+            // and expect our backends to respond in <100ms.
+            .connect_timeout(Duration::from_millis(500))
+            .read_timeout(Duration::from_millis(500))
```

Comment:
Let's also add options for this, please.

Reply:
Here's a suggestion on how to make it work:
- `ClientBuilder` doesn't build the client eagerly, instead it just prepares all the options
- Introduce a consuming `build` method that builds one client, which is also clonable.
- `for_organization` and `for_project` (along with `make_client` internally) become methods on `Client` that create a `ScopedClient`, like was discussed a while ago. 

Alternatively, you can also have it as `ClientOptions` -> `ClientBuilder` -> `Client` (instead of `ClientBuilder` -> `Client` -> `ScopedClient`), but I think that's less clear to the user.